src/index.html

```diff
@@ -1,27 +1,34 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="UTF-8">
-    <title>title</title>
-  <link rel="stylesheet" href="css/style.ss" type="text/css">
-  </head>
-  <body>
-      <div class="error" style="color:pink">
-          Looks like the styleguide is not being implemented correctly..
-      </div>
-      <div class="content">
-          <h1 class="title">
-              Hmm...
-          </h1>
-          <p class="text">
-              Something is wrong... <br> 
-              Looks like the Javascript code is not being executed. <br>
-              Fix me please.
-          </p>
-      </div>
-      <div class="success" style="color:white; position:absolute; top:580px">
-            Woohoo, CSS is working!
+    <title>VI Onboarding Fundamentals II</title>
+    <link rel="stylesheet" href="ssc/style.css">
+</head>
+
+<body>
+    <div class="card card--active card--secondary">
+        <div class="card__content">
+            <h1 class="card__title">
+                A wonderful serenity has taken possesion
+            </h1>
+            <p class="card__text">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            </p>
+            <a href="javascript:" class="card__button">Link button</a>
         </div>
-      <script type="module" src="js/mayn.js"></script>
-  </body>
-</html> 
\ No newline at end of file
+        <div class="card__imgBox">
+            <img src="https://dummyimage.com/800x600/c7c7c7/fff" alt="Image" class="card__img">
+        </div>
+        <!-- <div class="card__box">
+              <div class="card__innerBox">
+
+              </div>
+          </div> -->
+    </div>
+
+    <script type="module" src="js/main.js"></script>
```

Comment:
Please add all the variants to the page. The page is a showcase for all the different versions of the component, this is called a scaffolding. A scaffolding needs to have all the different versions of the component on it, so the editor or whoever uses the component after we build is, can see them all at the same time.